src/components/select_team_view.js

```diff
@@ -28,15 +28,15 @@ export default class SelectTeamView extends Component {
     }
 
     componentWillReceiveProps(nextProps) {
-        const {currentTeamId} = nextProps.teams;
+        const {currentTeamId} = nextProps.team;
```

Comment:
Feels like a hack?  Seems like routing should be handled by some outside control?  maybe I'm missing something.